doc/cljdoc.edn

```diff
@@ -26,6 +26,7 @@
   ["Additional Resources" {:file "doc/modules/ROOT/pages/additional_resources.adoc"}]
   ["About" {}
    ["History" {:file "doc/modules/ROOT/pages/about/history.adoc"}]
+   ["Contrib nREPL" {:file "doc/modules/ROOT/pages/about/contrib_nrepl.adoc"}]
```

Comment:
Probably it'd be better to just add this to the existing `History` page.

Reply:
Or rather the FAQ, given that all the info here is about upgrading from 0.2.x. In general we can probably remove all of this and nobody would notice this. You said it yourself - 7 years are a lot of time and I doubt anyone it still using nREPL 0.2 at this point.

Reply:
Done!